static/xref/index.html

```diff
@@ -16,7 +16,7 @@
       <div class="field">
         <label for="cite">Specification(s) short name</label>
         <div class="selections" data-for="cite"></div>
-        <input name="cite" placeholder="html,webidl,payment-request" />
+        <input name="cite" placeholder="html,webidl,infra..." />
```

Comment:
these placeholders get overridden as soon as `/xref/meta` resolves and we replace this `input` with `input[is]` though. but good to have it here.